parser/MyParser.java

```diff
@@ -221,23 +247,26 @@ public void extractTasks(String title, String url, String content, int outIndex)
       for (Task task : tasks) {
         numTasksAfterFiltering++;
         keptTasks.add(task.taskString);
-        bw.write(task.taskString);
-        bw.write("\n");
+        try {
+            PreparedStatement st = db.prepareStatement("INSERT INTO task (compute_index, task, search_result_content_id) VALUES (?, ?, ?)");
+            st.setInt(1, computeIndex);
+            st.setString(2, task.taskString);
```

Comment:
In the most recent revision, I did you one better and saved all of the lemmatized verbs and nouns in their own tables, so that we can aggregate them later and see which nouns and verbs are used the most frequently across search results.